src/components/ReviewRoutingSlip/ReviewRoutingSlipCashPayment.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-row class="d-flex justify-between" v-if="cashPayment">
-    <v-col cols="6">
+    <v-col :cols="cashPayment.paidUsdAmount > 0 ? 4: 6">
```

Comment:
make this as a getter function and can check here, since it re-using multiple places